apps/users/tests_users.py

```diff
@@ -0,0 +1,82 @@
+from django.test import TestCase
+
+from .admin import UserAdmin
+from .models import User
```

Comment:
```suggestion
from apps.users.admin import UserAdmin
from apps.users.models import User
```

Sempre utilizar o caminho inteiro para o módulo.